_posts/jamesma560/James-loop-exercises.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+author: jamesma560
+title: "James' Loop Exercises"
+---
+
+Loop exercise 1:
+
+<iframe src="https://trinket.io/embed/python/dd03c03880" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Loop exercise 2:
+
+<iframe src="https://trinket.io/embed/python/57aa16c429" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+
+I thought the first exercise was pretty easy. Not much to say about it. The second one gave me some trouble though. With the try/except 
+stipulation, coupled with the fact that I chose to use if statements to check to see if the user's input was the max or min value, keeping
+the indentation on the up and up was difficult. Another funny thing I noticed is that when I first ran my program, I kept getting 0's
+for the maximum and minimum values. And this bugged me for the longest time because I was FOR SURE that it wasn't the indentation. But for 
+some random reason in the midst of my frustration, I realized that I had written the max and min assignment statements backwards. The reason why
+I was getting 0's was because I had 'user_choice = maximum' as the assignment instead of 'maximum = user_choice". It was a silly mistake, but
+it was definitely an educational one. Finally, the biggest problem I had was figuring how to initialize the max and min values. At first, I 
```

Comment:
I get things exactly wrong all the time.  much better than getting them randomly wrong :)